daktari/checks/kubernetes.py

```diff
@@ -34,7 +34,7 @@ def check(self) -> CheckResult:
 
 
 def get_kubectl_version() -> Optional[VersionInfo]:
-    raw_version = get_stdout("kubectl version --client=true --short")
+    raw_version = get_stdout("kubectl version --client=true")
```

Comment:
Oh, erk - did they remove `--short`? I get an error trying to use it now myself :grimacing: